src/components/accordion/1_hiddenAttribute/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { useEffect, useRef, useState } from 'react';
+import cx from '../cx';
+import data from '../data';
+
+/* 참고: https://hiddenest.dev/accessible-accordion */
+const AccordionItem = ({
+  id,
+  title,
+  content,
+  current,
+  toggle,
+}: {
+  id: string;
+  title: string;
+  content: string;
+  current: boolean;
+  toggle: () => void;
+}) => {
+  const contentRef = useRef<HTMLDivElement>(null);
+  const triggerID = `accordion-trigger-${id}`;
+  const contentID = `accordion-content-${id}`;
+
+  useEffect(() => {
+    const $contentEl = contentRef.current;
+    if (!$contentEl) return;
+
+    if ($contentEl) {
+      $contentEl.addEventListener('beforematch', toggle);
+    }
+    return () => {
+      if ($contentEl) $contentEl.removeEventListener('beforematch', toggle);
+    };
+  }, [toggle]);
```

Comment:
오호... 이런 이벤트도 있군요 ㅎㅎ 덕분에 하나 알아갑니다!

그리고

```suggestion
    if (!$contentEl) return;

    $contentEl.addEventListener('beforematch', toggle);
    return () => $contentEl.removeEventListener('beforematch', toggle);
  }, [toggle]);
```
$container가 없는 상황알 앞에서 필터링해주기 때문에 이렇게 처리해도 무방해보여요!